.github/workflows/deployment.yml

```diff
@@ -0,0 +1,79 @@
+name: safe-gelato-relay-service
+
+on: [ push, pull_request ]
+
+jobs:
+
+  docker-publish-staging:
+    if: (github.event_name == 'push' && github.ref == 'refs/heads/main')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          # Number of commits to fetch. 0 indicates all history for all branches and tags.
+          fetch-depth: '0'
+      - name: Set Version
+        run: |
+          description="$(git describe --tags --always)"
+          echo "VERSION=${description:1}" >> $GITHUB_ENV
```

Comment:
Why do we need to set the version? This branch is triggered on `main`.

Also we don't use the `VERSION` environment variable at all in the service 🤔